deepeval/models/hallucination_model.py

```diff
@@ -0,0 +1,18 @@
+import os
+from deepeval.singleton import Singleton
+from sentence_transformers import CrossEncoder
+from deepeval.models.model_map import model_map, name_to_card
+
+
+class HallucinationModel(metaclass=Singleton):
+    def __init__(self, model_name: str = "vectara-hallucination"):
+        # We use a smple cross encoder model
+        os.environ["TOKENIZERS_PARALLELISM"] = "false"
+        model_name = name_to_card(model_name)
+        self.model = CrossEncoder(model_name)
+
+    def score(self, source: str, prediction: str):
```

Comment:
No need for this, scoring logic should be in scoring.py and the model module can be reserved for handling the initialization of a model instead.